src/utils/executables.cmake

```diff
@@ -53,6 +53,7 @@ TICCalculator
 TransformationEvaluation
 TopPerc
 XMLValidator
+XFDR
```

Comment:
F before M 
could you also change the order of TopPerc in the file so it is sorted